pkg/controller/servicebindingrequest/custom_env_path.go

```diff
@@ -0,0 +1,41 @@
+package servicebindingrequest
+
+import (
+	"bytes"
+	"github.com/redhat-developer/service-binding-operator/pkg/apis/apps/v1alpha1"
+	"text/template"
+)
+
+type CustomEnvPath struct {
+	EnvMap []v1alpha1.EnvMap
+	Cache map[string]interface{}
+}
+
+func NewCustomEnvPath(envMap []v1alpha1.EnvMap, cache map[string]interface{}) *CustomEnvPath {
+	return &CustomEnvPath{
+		EnvMap: envMap,
+		Cache:  cache,
+	}
+}
+
+func (c *CustomEnvPath) Parse() (map[string][]byte,error) {
```

Comment:
Since we tend to use `map[string]interface{}` in a number of places, should we give it a specific `type`?

Reply:
Yeah, good idea.

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/159/commits/39aa0717f996e734b697c629687656a36e74bacc